### ISSUE TITLE
[feature] can't install config from a remote .tar.gz

### ISSUE THREAD

**Issue by dvirtz** [conan-io/conan#9382]
I can install config from a local `.tar.gz` file but not from a remote one.
The problem seems to be that the file is saved with a `.zip` extension and not with its original one.

### Environment Details (include every applicable attribute)
  * Conan version: 1.37.2

### Steps to reproduce (Include if Applicable)
`conan config install <url to .tar.gz>`

### Logs (Executed commands with output) (Include/Attach if Applicable)
```
> conan config install <hidden>/conan-config.tar.gz
Trying to download  <hidden>/conan-config.tar.gz
Downloading config.zip completed [70.00k]                                                

ERROR: Failed conan config install: Error while installing config from https://artifactory.ouroath.com/artifactory/uplynk-generic/conan-config/ee9475be/conan-config.tar.gz
File is not a zip file
```

**Comment by lasote** [comment]
Yes, it is limited to `zip`. It is not a bug but a missing feature ;)

Thanks!

**Comment by el-g-1** [comment]
I'm working on it.

**Comment by memsharded** [comment]
Merged. Will be released next 1.41.
